tests/frontend/unit/ParticipationPhases.spec.js

```diff
@@ -0,0 +1,89 @@
+import { createLocalVue } from '@vue/test-utils'
+import ParticipationPhases from '@DpJs/components/procedure/basicSettings/ParticipationPhases'
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+
+describe('ParticipationPhases', () => {
+  const componentsPropsData = {
+    autoswitchHint: 'Here comes a tooltip Hint',
+    fieldName: 'some_name',
+    helpText: 'Some Help Text',
+    labelText: 'I am a Label',
+    participationPhases: ['first', 'second', 'third'],
+    permissionMessage: 'Some Message',
+    phaseOptions: [{
+      label: 'My Label C',
+      permissionset: 'read',
+      value: 'first'
+    }, {
+      label: 'My Label A',
+      permissionset: 'write',
+      value: 'fourth'
+    }, {
+      label: 'My Label B',
+      permissionset: 'hidden',
+      value: 'fifth'
+    }],
+    initSelectedPhase: 'first',
+    iterator: {
+      label: 'Iterator Label',
+      name: 'iterator_input_name',
+      tooltip: 'Tooltip text for the itereator',
+      value: '1'
+    }
+  }
+
+
+  it('should be an object', () => {
+    expect(typeof ParticipationPhases).toBe('object')
+  })
+
+  it('should be named ParticipationPhases', () => {
+    expect(ParticipationPhases.name).toBe('ParticipationPhases')
+  })
+
+  it('should set the "inParticipation"-State as expected', async () => {
```

Comment:
I think tests should also document how a component is supposed to work, so I find it useful to say it explicitly (rather than 'as expected'. what is expected?), e.g. 'sets `inParticipation` to true if the selected Phase is a participation phase'. I would also test only one case in a single test, so I'd add separate tests for when `inParticipation` should be `false`.